include/vctr/Containers/VctrBase.h

```diff
@@ -303,7 +308,7 @@ class VctrBase : private Config,
 
         const auto increment = (stop - start) / double (num - size_t (includeEnd));
 
-        [[maybe_unused]] const auto isIntegerIncrement = increment - std::floor (increment) == ElementType (0);
+        [[maybe_unused]] const auto isIntegerIncrement = increment - gcem::floor (increment) == ElementType (0);
```

Comment:
GCEM is only an optional dependency, so every code using it should provide a fallback solution for cases when GCEM is not available